### DISCUSSION TITLE
How can I type any text without taking care of special characters?

### DISCUSSION THREAD

**Discussion by amalgame21** [nvim-neorg/neorg#1074] [Q&A]
For example, I have a note in txt file about the usage of vim, which contains a lot of stuff like `.\{-}`, `\ze` etc where some of the characters need to be escaped.
How can I make sure that all the actual characters can be displayed without manually escaping them?
I know the `@code @end` can somehow achieve this but I think it is not intended for this?
And seems that it can not be applied on the Headings, etc.
Thanks!

**Comment by vhyrro** [comment]
It's been in my plans for a while to implement a custom formatting and paste handler in Neovim to escape special characters but that's too low on the priority list for me to handle right now. Just know that I am aware of the issue! :)
